partner_code_program_sale/tests/test_partner_code_program_sale.py

```diff
@@ -0,0 +1,35 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestPartnerCodeProgramSale(TransactionCase):
```

Comment:
Can you add the test where you create a sale and check if the sale has the partner's code program?